tests/travis/main.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash